test/colony-funding.js

```diff
@@ -62,26 +62,26 @@ contract("Colony Funding", accounts => {
       let colonyRewardPotBalance = await colony.getPotBalance(0, otherToken.address);
       let colonyPotBalance = await colony.getPotBalance(1, otherToken.address);
       let colonyTokenBalance = await otherToken.balanceOf(colony.address);
-      assert.equal(colonyTokenBalance.toNumber(), 100);
-      assert.equal(colonyPotBalance.toNumber(), 0);
-      assert.equal(colonyRewardPotBalance.toNumber(), 0);
+      expect(colonyTokenBalance).to.eq.BN(100);
+      expect(colonyPotBalance).to.eq.BN(0);
+      expect(colonyRewardPotBalance).to.eq.BN(0);
```

Comment:
Can we use the full potential of `bn-chai` and do `to.be.zero`?

Reply:
Yes! I was just circling back to make those updates.